python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -180,6 +180,7 @@ def __init__(self, registry, *args, **kwargs):
         super(RegistryFlusher, self).__init__(*args, **kwargs)
         self.registry = registry
         self._stop = threading.Event()
+        self.logger = getLogger()
```

Comment:
They will be indistinguishable from each other as they will have the same name I believe and in fact might even just be references to the same object.